src/AI_Sideloader/AI.Sideloader.ListLoader.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace Sideloader.ListLoader
+{
+    internal static partial class Lists
+    {
+        // internal main game furniture data
```

Comment:
Move above to be a comment of class Lists or it can get lost if someone does any refactoring